index.html

```diff
@@ -23,212 +23,154 @@ <h1>I’m Tshepo. <br> Glad to see <br> you!</h1>
       <a href="#" class="Connect">LET’S CONNECT</a>
       
       <ul class="Social">
-          <li><a href="#"><img src="image/Vector5.png" alt=""> </li>
-          <li><a href="#"><img src="image/Linkedin icon (1).png" alt=""></a></li>
-          <li><a href="#"><img src="image/Vector1.png" alt=""></a></li>
-          <li><a href="#"><img src="image/Vector2.png" alt=""></a></li>
-          <li><a href="#"><img src="image/Vector3.png" alt=""> </a></li>
-      
+          <li><a href="#"><img src="image/Vector5.png" alt="Vector"> </li>
+          <li><a href="#"><img src="image/Linkedin icon (1).png" alt="Linkedin"></a></li>
+          <li><a href="#"><img src="image/Vector1.png" alt="Vector"></a></li>
+          <li><a href="#"><img src="image/Vector2.png" alt="Vector"></a></li>
+          <li><a href="#"><img src="image/Vector3.png" alt="Vector"> </a></li>
       </ul>
   </div>    
-  
-</div>
-
-
-
-
-
-
-   
-
+  </div>
 
-    <div class="Container2">
-      <img class="Snapshoot2" src="image/SnapshootD3.png" alt="">
+   <section class="Container2">
```

Comment:
- **[OPTIONAL]** In the work section all the cards have a margin, please add it to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Tshepo45/Mobile-Finish/assets/58642949/70e24bc8-865e-4e60-ae2b-9131da4b4353) | ![image](https://github.com/Tshepo45/Mobile-Finish/assets/58642949/d30ed016-1685-4929-9a32-89ea7a2acbb8) |